server/server_controller.go

```diff
@@ -493,3 +493,11 @@ func (c *ServerController) CheckServerInstalled(id int) (bool, error) {
 }
 
 //endregion
+
+func (c *ServerController) GetServerStartupCommand(id int) string {
```

Comment:
also return the error to the frontend like (string, error)
look at this for example:
```go
func (c *ServerController) GetServerStatus(id int) (bool, error) {
	server, exists := c.Servers[id]
	if !exists {
		err := fmt.Errorf("error getting server status " + strconv.Itoa(id) + ": server does not exist in map")
		runtime.LogError(c.ctx, err.Error())
		return false, err
	}

	status := server.IsServerRunning()
	return status, nil
}
```